app/controllers/api/v4/questionnaires_controller.rb

```diff
@@ -30,9 +30,14 @@ def force_resync?
     locale_modified? || resync_token_modified?
   end
 
+  def current_facility_processed_since
+    return Time.new(0) if force_resync?
+    process_token[:current_facility_processed_since].try(:to_time) || Time.new(0)
+  end
+
   def response_process_token
     {
-      other_facilities_processed_since: processed_until(other_facility_records) || other_facilities_processed_since,
```

Comment:
We moved to current_facilities_processed_since from other_facilities_processed_since because the layouts get synced for current facility.

other_facilities_processed_since would have confused future devs reading this code. 

Reason we were doing other facilities before was default `current_facility_processed_since` method is doing too many things. We've overridden that method.